packages/a11y-testing/src/facades/ComponentTestFacade.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { Props, PropValue, TestFacade } from '@fluentui/a11y-testing';
+import { mountWithProviderAndGetComponent } from './withProvider';
+
+export class ComponentTestFacade implements TestFacade {
+  private actual: any;
+  private renderedComponent: any;
+  private onClickExecuted: boolean;
+
+  constructor(private Component: React.FC, private props: Props = {}) {
+    props.onClick = () => {
+      this.onClickExecuted = true;
+    };
+
+    // reset body
+    document.body.innerHTML = '';
+    const container = document.createElement('div');
+    document.body.appendChild(container);
+
+    ReactDOM.render(<Component {...props} />, container);
+    this.actual = container.lastChild;
+
+    // we need to render it in this way because using simulate function to fire mouse/keyboard event
+    this.renderedComponent = mountWithProviderAndGetComponent(Component, <Component {...props} />);
```

Comment:
I was refering to trying to remove this `mountWithProviderAndGetComponent` and just render the `Component` itself.

Reply:
yes thanks, replaced with line:
` this.renderedComponent = mount(<Component {...props} />).find(Component);`